src/controllers/RaceController.js

```diff
@@ -0,0 +1,74 @@
+import Cars from "../models/Cars.js";
+import Winner from "../models/Winner.js";
+import CarNameValidator from "../validators/CarNameValidator.js";
+import TryCountValidator from "../validators/TryCountValidator.js";
+
+class RaceController {
+  #inputView;
+  #outputView;
+
+  constructor(views) {
+    this.#inputView = views.inputView;
+    this.#outputView = views.outputView;
+  }
+
+  async race() {
+    const parseCarNames = await this.#initCarNames();
+    const parseTryCount = await this.#initTryCount();
+
+    const cars = new Cars(parseCarNames);
+
+    this.#processRacing(cars, parseTryCount);
+    this.#processWinner(cars);
+  }
+
+  async #initCarNames() {
+    while(true) {
+      try {
+        const carNames = await this.#inputView.getCarNames();
+        const parseCarNames = carNames.split(",").map((carName) => carName.trim());
+        new CarNameValidator().valiateNames(parseCarNames);
+  
+        return parseCarNames;
+      } catch (error) {
+        console.log(error);
+      }
+    }
+  }
+
+  async #initTryCount() {
+    while(true) {
+      try {
+        const tryCount = await this.#inputView.getTryCount();
+        const parseTryCount = Number(tryCount);
+        new TryCountValidator().validateNumber(parseTryCount);
+  
+        return parseTryCount;
+      } catch (error) {
+        console.log(error);
+      }
+    }
+  }
+
+  #processRacing(cars, parseTryCount) {
```

Comment:
```suggestion
  #processRacing(cars, parsedTryCount) {
```
변수명이 동사로 시작하면 이를 함수로 오해하기 쉽습니다. 함수가 아닌 변수는 이를 지양해주세요.

Reply:
parseTryCount라고 하면 함수명으로 오해할 소지가 있는 것 같아요. 다시보니 크리스가 언급해주신대로 parsedTryCount로 명명하는 것이 올바른 것 같습니다.

다음 단계에서도 이 부분 주의하겠습니다 !! 🫡 세세하게 리뷰해주셔서 감사해요